src/main/java/domain/Board.java

```diff
@@ -0,0 +1,77 @@
+package domain;
+
+import domain.chessPiece.ChessPiece;
+import domain.hurdlePolicy.HurdlePolicy;
+import domain.path.Path;
+import domain.position.ChessPiecePositions;
+import domain.position.ChessPosition;
+import domain.score.Score;
+import domain.type.ChessTeam;
+
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+
+public class Board {
+
+    private final ChessPiecePositions chessPiecePositions;
+    private final Map<ChessTeam, Score> scores = new EnumMap<>(ChessTeam.class);
+
+    public Board(final ChessPiecePositions chessPiecePositions) {
+        this.chessPiecePositions = chessPiecePositions;
+        scores.putAll(Map.of(
+                ChessTeam.RED, Score.zero(),
+                ChessTeam.BLUE, Score.zero()
+        ));
```

Comment:
아래와 같이 생성자를 통해 scores 필드를 바로 초기화해볼 수도 있을 것 같아요.

```suggestion
    private final Map<ChessTeam, Score> scores;

    public Board(final ChessPiecePositions chessPiecePositions) {
        this.chessPiecePositions = chessPiecePositions;
        this.scores = Map.of(
                ChessTeam.RED, Score.zero(),
                ChessTeam.BLUE, Score.zero()
        );
    }
```